packages/components/src/styles/components/side-nav/main.scss

```diff
@@ -7,7 +7,23 @@
 // SIDE-NAV
 //
 
+// TODO: Create app-header tokens:
+:root {
+  --token-app-header-height: 60px;
+  --token-app-header-home-link-padding: 4px;
+}
+
+:root {
+  // TODO: Create sidenav token
+  --hds-app-sidenav-headerless-width-minimized: 16px;
+}
+
 .hds-side-nav {
+  // set narrower collapsed width when the header is empty
+  &:has(.hds-side-nav__wrapper-header:empty) {
+    --hds-app-sidenav-width-minimized: var(--hds-app-sidenav-headerless-width-minimized);
```

Comment:
@didoo Should I create a new JSON token for `--hds-app-sidenav-headerless-width-minimized`?

Or should I just set the value directly here?
```
--hds-app-sidenav-width-minimized: 16px;
```

Reply:
I decided to set the value directly for phase 1 at least. (Can reconsider in phase 2)